### ISSUE TITLE
live.asp.net is down

### ISSUE THREAD

**Issue by dsteinweg** [aspnet/live.asp.net#151]
https://live.asp.net/ shows:
```
Error.
An error occurred while processing your request.
© 2015 Microsoft. All rights reserved.

Privacy Statement  Terms of Use  Source Code  Issue Tracker
```

**Comment by alexwiese** [comment]
Looks like it's back online now.

**Comment by dsghi** [comment]
Down again :(

**Comment by chriswoodward** [comment]
Site is down Wednesday morning here in the UK ... 

![firefox screenshot](https://user-images.githubusercontent.com/3061603/51600218-69122e80-1ef9-11e9-9995-bbf3ec124268.png)

**Comment by terrimorton** [comment]
We tracked this down to a deleted video being present in the playlist.  We removed the video from the playlist, which brought the website back up.  We filed 2 issues to gracefully handle this situation as well as to handle when future events are found in the playlist.

[Skip deleted videos in the playlist](https://github.com/aspnet/live.asp.net/issues/155)
[Skip future events in the playlist](https://github.com/aspnet/live.asp.net/issues/156)
